packages/runtime/src/runtime/bots/bot-service.ts

```diff
@@ -33,6 +32,7 @@ const debug = DEBUG('services:bots')
 
 @injectable()
 export class BotService {
+  public botImported?: (botId: string) => void
```

Comment:
That sounds like a boolean.. why not wipeBotCache?  Also, the method is async, it should be Promise<void>

Reply:
Good points. I renamed it to onBotImported and made it async